src/stdune/csexp.mli

```diff
@@ -0,0 +1,15 @@
+type t = Sexp.t
+
+exception Parse_error of string
+
+val to_buffer_canonical : buf:Buffer.t -> t -> unit
```

Comment:
The `_canonical` suffix feels a bit redundant with the `C` of `Csexp`. We can dop it